_maps/map_files/cerestation/cerestation.dmm

```diff
@@ -24861,7 +24861,8 @@
 	department = "Cargo Bay";
 	departmentType = 2;
```

Comment:
this departmentType should also be 5 I think?

Reply:
I fixed it, its now set to 5